src/seeders/20200904133535-User.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+
+    await queryInterface.bulkInsert ('Users', [{
+      email: 'olusola@gmail.com',
```

Comment:
in all your seeders, have the ids included there, even the foreign ids... you can get fake ids(uuid) here

https://www.uuidgenerator.net/version4